app/views/admin/configs/show.html.erb

```diff
@@ -24,8 +24,8 @@
       <div id="getStartedBodyContainer" class="<%= VerifySetupCompleted.setup_completed? ? "hide collapse" : "show expand" %> p-3" aria-labelledby="siteConfigHeader">
         <%= form_for(SiteConfig.new, url: admin_config_path) do |f| %>
 
+          <!-- we need to list the config as separate fields if the data structure is a Hash -->
```

Comment:
Moved this comment up because it was rendering every time we iterated over a config, which made it hard to read the DOM 😕

Reply:
**suggestion (non-blocking):** How about we leave it where it is (because it really relates to the `if` statement), but turn it into an ERb comment that doesn't show up in the generated HTML?  

```ruby
<% VerifySetupCompleted::MANDATORY_CONFIGS.each do |config_key| %>
  <%# we need to list the config as separate fields if the data structure is a Hash %>
  <% if SiteConfig.public_send(config_key).is_a?(Hash) %>
   ...
<% end %>
```

Reply:
@citizen428 that's a great suggestion and compromise -- i like it!